### ISSUE TITLE
Mac - Font in windowed main GUI "fades" while mouse moving

### ISSUE THREAD

**Issue by StusBrainDump** [BlitterStudio/amiberry#1356]
**Describe the bug**
Having launched either 572 or 632 (apple silicon or x86_64 version) on an M3 Max Macbook Pro (macOS Sonoma 14.5), moving the mouse in the GUI causes text to become "thinner" and lighter. When you stop moving the mouse, about a second later the font returns to it's normal thickness.

**To Reproduce**

Launch Amiberry 572 or 632
Observe thickness of default font in main UI
Move mouse around UI (don't need to click, just swirl the mouse within the bounds of the UI)
Font "fades" and becomes thinner
Stop moving mouse and shortly after the font returns to normal.

This behaviour is not observed if you move the mouse outside the bounds of the app, or if you make the app fullscreen.

**Expected behavior**
Font should remain same size/thickness regardless.

**Screenshots**

![ThinFont](https://github.com/BlitterStudio/amiberry/assets/119980446/909a9552-45bd-466d-b3b6-152e26a0e350)

![ThickFont](https://github.com/BlitterStudio/amiberry/assets/119980446/8d7392ab-5f09-4355-869d-5a33ea6d7dba)

**Comment by midwan** [comment]
That definitely doesn't happen on any other platform here, so I have to assume it's some macOS-specific glitch.
Did this start happening after a certain version, or was it always like that?

**Comment by StusBrainDump** [comment]
Just checked .... happens in 571 but doesn't in 570.

**Comment by midwan** [comment]
@StusBrainDump Can you check if this one below makes a difference, please?

[amiberry-macOS-64bit-m1.zip](https://github.com/user-attachments/files/15822116/amiberry-macOS-64bit-m1.zip)

**Comment by StusBrainDump** [comment]
Was just downloading commits and testing:

amiberry-1e081a37285f89ff2dfbf8fc8e93f80f9c5336ad is the first commit to exhibit the issue
amiberry-573073fd61356817333e14a50e223a9426b25c05 (prior to above) does NOT exhibit the issue.

**Comment by StusBrainDump** [comment]
> @StusBrainDump Can you check if this one below makes a difference, please?
> 
> [amiberry-macOS-64bit-m1.zip](https://github.com/user-attachments/files/15822116/amiberry-macOS-64bit-m1.zip)

This works fine, no font issues. I assume this is a newer build of 632 as the 632 I already have DOES have the font issue.

**Comment by midwan** [comment]
Yes, this is a newer build. Thanks for confirming, I've bumped the version and will push this fix to both `master` and `preview`. 

The cause was the HighDPI flag, that was removed for a test earlier, and I never brought it back. Until now. :)

**Comment by midwan** [comment]
Fixed with d55fb380b733828d9e31809f352710bb406ff9a0

**Comment by StusBrainDump** [comment]
> Yes, this is a newer build. Thanks for confirming, I've bumped the version and will push this fix to both `master` and `preview`.
> 
> The cause was the HighDPI flag, that was removed for a test earlier, and I never brought it back. Until now. :)

Thanks :)

Legend as always!
